common.py

```diff
@@ -69,9 +69,11 @@ def __init__(self, dictionary):
     def write(self):
         """This function writes the data to elastic search"""
         self.dictionary = json.dumps(self.dictionary)
-        requests.post('http://elasticsearch2.gridpp.rl.ac.uk:9200/logstash' +
-                       '-gridtools-metrics-'
-                       + date + '/apel/', data=self.dictionary)
+        requests.post('%s/%s-%s/metric_data/' %
+                      ('http://elasticsearch2.gridpp.rl.ac.uk:9200',
```

Comment:
Why are you using string formatting for more than just the variable `date`?

Reply:
Presumably the URL and index should be constants?

Reply:
I am using string formatting to avoid long lines

Reply:
If it's just for the line length you can wrap strings inside brackets. Just end the line using whichever quote mark you're using and start the next line with another. I've used it quite a lot in unit tests with large string data: https://github.com/apel/apel/blob/dev/test/test_sge.py#L39

https://www.python.org/dev/peps/pep-0008/#maximum-line-length
> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses.

Reply:
changed